backend/src/schema/resolvers/posts.spec.js

```diff
@@ -140,45 +159,88 @@ describe('Post', () => {
       ).resolves.toMatchObject(expected)
     })
 
-    it('by emotions', async () => {
+    describe('by emotions', () => {
+      const postQueryFilteredByEmotions = gql`
+        query Post($filter: _PostFilter) {
+          Post(filter: $filter) {
+            id
+            emotions {
+              emotion
+            }
+          }
+        }
+      `
+
+      it('filters by single emotion', async () => {
+        const expected = {
+          data: {
+            Post: [
+              {
+                id: 'happy-post',
+                emotions: [{ emotion: 'happy' }],
+              },
+            ],
+          },
+        }
+        await user.relateTo(happyPost, 'emoted', { emotion: 'happy' })
+        variables = { ...variables, filter: { emotions_some: { emotion_in: ['happy'] } } }
+        await expect(
+          query({ query: postQueryFilteredByEmotions, variables }),
+        ).resolves.toMatchObject(expected)
```

Comment:
Here you omit the `expect.arrayContaining` :+1:

Reply:
yes, we wanted to use this for the `multiple emotions` test as well, but it failed as I stated :point_up: